governance/votes/Executive vote - January 24, 2024.md

```diff
@@ -0,0 +1,141 @@
+---
+title: Template - [Executive Vote] Stability Fee Changes, PSM-PAX-A Debt Ceiling Reduction and DC-IAM Disabling, RETH-A Offboarding Parameters Finalization, SBE Parameter Changes, HV Bank DAO Resolution Approval, AVC Members Compensation, Spark Proxy Spell - January 24, 2024
```

Comment:
```suggestion
title: Template - [Executive Vote] Stability Fee Changes, PSM-PAX-A Debt Ceiling Reduction, RETH-A Offboarding Parameters Finalization, SBE Parameter Changes, HV Bank DAO Resolution, Q4 2023 AVC Member Compensation, Spark Proxy Spell - January 24, 2024
```
I always include the compensation period in the title to make it easier to search for in the future